### PR TITLE
Table printer with right justified

### PR DESCRIPTION
**Pull request by shafqat-muneer**

### CODE REVIEW

File: TablePrinter.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# TablePrinter.py - Take list of lists of strings and display
+# it in well-organized table.
+
+
+'''
+For practice, write a program that does the following.
+
+Table Printer
+Write a function named printTable() that takes a list of lists of strings and displays it in a well-organized table with each column right-justified. Assume that all the inner lists will contain the same number of strings.For example, the value could look like this:
```
Comment (uzairr) [inline TablePrinter.py]: split it into multiple lines, follow the command-line convention i.e. each line should be of 80 characters.

File: TablePrinter.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        print()
+    
+
+def findMaxLength(listItems):
+    # It will first find maximum size string
+    # After taking that maximum size string, we are taking lenght of that string
+    return len(max(listItems, key=len))
+        
+
+printTable()
```
Comment (uzairr) [inline TablePrinter.py]: This should only be executed if this module is run as a main module. i.e.
if __name__ =='__main__':
    print_table()

Other than that please follow pep8 conventions especially for naming variables and methods
https://www.python.org/dev/peps/pep-0008/

**Review by uzairr — COMMENTED** [review]
Spent some time to review pep8 conventions.

File: TablePrinter.py (hunk 1 above)
Comment (shafqat-muneer) [reply TablePrinter.py]: Done

**Pull request merged**